client/signup/steps/site-options/index.tsx

```diff
@@ -55,6 +67,7 @@ export default function SiteOptionsStep( props: Props ): React.ReactNode {
 			skipLabelText={ translate( 'Skip this step' ) }
 			isHorizontalLayout={ true }
 			{ ...props }
+			goToNextStep={ skipStep }
```

Comment:
We can just use `defaultDependencies` as followed because it would submit the `defaultDependencies` when you click the skip button. Otherwise, the `submitSignupStep` would be called twice.
```js
defaultDependencies={ {
  siteTitle: '',
  tagline: '',
} }
```

Reply:
ahh nice 👍